src/commands/login.mjs

```diff
@@ -39,16 +39,21 @@ function buildLoginCommand(yargs) {
     user: {
       alias: "u",
       type: "string",
-      description: "User to log in as",
+      description: "User to log in as.",
       default: "default",
-      group: "login Options:",
+      group: "Login options:",
```

Comment:
```suggestion
```

This arg shouldn't have a group (so it gets grouped with `--help` and `--version`).

Reply:
Gotcha. This was just a capitalization change, but I'll remove it in a separate PR. Thanks!

Reply:
PR: https://github.com/fauna/fauna-shell/pull/480